src/metemctl_misp.py

```diff
@@ -30,13 +32,16 @@
 if __name__ == '__main__':
 
     args = docopt(__doc__)
+    print(args)
     
     config = configparser.ConfigParser()
     config.read(CONFIG_INI_FILEPATH)
     
-    if args['--open']:
+    if args['open']:
         browser = config['general']['browser_path']
         url = config['general']['misp_url']
+        if args['--url']:
+            url = args['--url']
         exit(call([browser, url]))
     else:
         exit("Options are not set. See 'metemctl misp --help'.")
```

Comment:
Optionsの動作判定の廃止されたため、"Options are not set"は変更する必要あり(現状このコードは参照されない)。
```suggestion
        exit("Invalid command. See 'metemctl misp --help'.")
```